search/includes/classes/class-search.php

```diff
@@ -43,7 +43,7 @@ protected function setup_constants() {
 		}
 
 		if ( ! defined( 'EP_HOST' ) && defined( 'VIP_ELASTICSEARCH_ENDPOINTS' ) && is_array( VIP_ELASTICSEARCH_ENDPOINTS ) ) {
-			$host = VIP_ELASTICSEARCH_ENDPOINTS[ 0 ];
+			$host = self::load_balance_endpoints( VIP_ELASTICSEARCH_ENDPOINTS );
```

Comment:
Thoughts on making this and the others an instance method (vs. static)? They don't technically need anything related to the instance, but also don't need to be called statically, so keeping them as instance methods is probably more consistent with the rest of the class.

Reply:
Sure, makes sense.